src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1883,14 +1644,7 @@ class SaagieClient {
                 }
                 return listPipelines
             }
-        } catch (InvalidUserDataException invalidUserDataException) {
-            throw invalidUserDataException
-        } catch (GradleException stopActionException) {
-            throw stopActionException
-        } catch (Exception exception) {
-            logger.error('Unknown error in projectsImport')
-            throw exception
-        }
+        }, 'Unknown error in projectsImport')
```

Comment:
Maybe add the name of the function too, like
'Unknown error in task projectsImport, function getPipelineListByNameAndId'

It will be more understandable and allow us to know where to start debugging

Reply:
This is a mistake of my belalf of to log the `projectsImport` instead of  `getPipelineListByNameAndId` but uselly it s one methoded that is concerned. But i did add `potentialFunctionName` to add potential function that may cause the Exception